package-lock.json

```diff
@@ -0,0 +1,347 @@
+{
+  "name": "indigo-web",
+  "version": "5.2.1",
```

Comment:
This is generated with node 17, which we're not using yet.

You'll need to delete this, ensure you're using node 16 (which is still the recommended version as per https://nodejs.org/en/) and then re-generate the lock file.

Reply:
ok. Hmm, its bit strange. I checked in my terminal and I am using node 16. However, nevertheless I did follow your advice anyway just to be doubly sure. My actions:
- `nvm use 16`
- delete `package-lock.json`
- remove `npm install`
Still produces the same lock file (no difference in git)🤔